recipe/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-mkdir build
-cd build
+mkdir build_release
+cd build_release
```

Comment:
Right, makes sense. Mac has similar issues so it is good to do the same thing here.